src/Components/Counter.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+
+function Counter({ fruitName, fruitImage, buttonType }) {
+
+    const [fruitCounterValue, setFruitCounterValue] = useState(0);
+
+    return (
+        <>
+            <article>
+                {fruitImage}
+                <h3>{fruitName}</h3>
+                <button
+                    type={buttonType}
+                    onClick={() => setFruitCounterValue(fruitCounterValue - 1)}
+                    disabled={fruitCounterValue === 0}
+                >
+                    -
+                </button>
+                <p>{fruitCounterValue}</p>
+                <button
+                    type={buttonType}
+                    onClick={() => setFruitCounterValue(fruitCounterValue + 1)}
+                >
+                    +
+                </button>
+                <button
+                    type="button"
+                    onClick={setFruitCounterValue(fruitCounterValue === 0)}
```

Comment:
Alle buttons zien er goed uit. Het enige probleem waar je tegen aan gaat lopen op deze manier, is dat je maar 1 state hebt gebruikt.

Zodra je het component meerdere keren gaat gebruiken, zal je tegen problemen aan lopen.

Elke keer als je op bijvoorbeeld de button klikt waar je de state met 1tje omhoog gaat, zal de state bij alle buttons ook met 1 omhoog gaan. Dit komt dus omdat je maar 1 state hebt gemaakt. Je zult dus eigenlijk voor elk component een nieuwe state willen maken, zodat per component individueel de waarde kan worden opgeslagen in state.